### PR TITLE
Add CRB for vrutkovs to access prowjobs

### PR DESCRIPTION
**Pull request by vrutkovs**
This is required to debug cluster-bot features and bugfixes

### CODE REVIEW

File: clusters/app.ci/assets/vrutkovs-pj-access.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vrutkovs-prowjob-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prowjob-create
```
Comment (petr-muller) [inline clusters/app.ci/assets/vrutkovs-pj-access.yaml:8]: I guess you'd want one to `prowjob-read` too? Feel free to unhold if not.

/approve
/hold

**Review by petr-muller — APPROVED** [review]

File: clusters/app.ci/assets/vrutkovs-pj-access.yaml (hunk 1 above)
Comment (vrutkovs) [reply clusters/app.ci/assets/vrutkovs-pj-access.yaml:8]: `cluster-bot` needs permissions to create new prowjobs

Comment (vrutkovs) [reply clusters/app.ci/assets/vrutkovs-pj-access.yaml:8]: Ah, correct. Added CRB for that too

**Review by petr-muller — APPROVED** [review]

**Pull request merged**